data/authors.csv

```diff
@@ -1,7 +1,7 @@
-﻿email;firstname;lastname

```

Comment:
did VS code automatically change this file? Looks like it striped out those BOM whitespace characters.

Reply:
Not sure what you mean.